reference/constraints/Negative.rst

```diff
@@ -0,0 +1,106 @@
+
+==========  ===================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Options     - `groups`_
+            - `message`_
+            - `payload`_
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\Negative`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\LesserThanValidator`
+==========  ===================================================================
+
+Basic Usage
+-----------
+
+The following constraint ensure that:
+
+* the ``withdraw`` of a  bankaccount ``TransferItem`` is a negative number (lesser than zero)
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/Entity/TransferItem.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class TransferItem
+        {
+            /**
+             * @Assert\Negative
+             */
+            protected $withdraw;
+        }
+
+    .. code-block:: yaml
+
+        # config/validator/validation.yaml
+        App\Entity\TransferItem:
+            properties:
+                withdraw:
+                    - Negative
+
+    .. code-block:: xml
+
+        <!-- config/validator/validation.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping https://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
+
+            <class name="App\Entity\TransferItem">
+                <property name="withdraw">
+                    <constraint name="Negative"></constraint>
+                </property>
+            </class>
+        </constraint-mapping>
+
+    .. code-block:: php
+
+        // src/Entity/TransferItem.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Mapping\ClassMetadata;
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class TransferItem
+        {
+            public static function loadValidatorMetadata(ClassMetadata $metadata)
+            {
+                $metadata->addPropertyConstraint('withdraw', new Assert\Negative());
+            }
+        }
+
+Available Options
+-----------------
+
+.. include:: /reference/constraints/_groups-option.rst.inc
+
+message
+~~~~~~~
+
+**type**: ``string`` **default**: ``This value should be negative.``
+
+The default message supplied when the value is not less than zero.
+
+You can use the following parameters in this message:
+
+=============================  ================================================
+Parameter                      Description
+=============================  ================================================
+``{{ compared_value }}``       Always zero
+``{{ compared_value_type }}``  The expected value type
```

Comment:
Not sure if we should include `compared_value` and `compared_value_type` here. They are available, but it doesn't make so much sense to me to use them. WDYT?

Reply:
I am not sure if we haven't added such (if available) parameters for other constraints.

Please help @xabbuh @javiereguiluz

Reply:
Even if they won't be used frequently, for consistency with the other constraints I'd prefer to keep them. We always try to document all parameters. Thanks.